data/lang/ui-core/en.json

```diff
@@ -283,6 +283,14 @@
     "description": "tooltip description in settings menu",
     "message": "Lower gives better performance, at cost of visual aesthetics"
   },
+  "DISABLE_SCREENSHOT_INFO": {
+    "description": "",
+    "message": "Disable Screenshot Info"
+  },
+  "DISABLE_SCREENSHOT_INFO_DESC": {
+    "description": "",
+    "message": "Check this to show nothing, not even System information, in screenshot mode (toggle HUD mode)."
+  },
```

Comment:
Nitpick, but not sure I like the "Check this to " part. A bit redundant.

Instead maybe have:
```
"When in screenshot mode (toggle HUD mode), hide all displayed info, including System information"
```
or
```
"Hide all displayed info, including system coordinates, when in screenshot mode (toggle HUD mode)"
```
Or a combination of the two. Or neither. Up to you.